src/main/java/frc/robot/Controllers.java

```diff
@@ -215,9 +215,19 @@ public static double getControllerAxis(
     }
   }
 
+  private static GenericEntry browningOutRumble =
+      Shuffleboard.getTab("Driver")
+          .add("Rumble if battery voltage is below 10.5v", true)
+          .getEntry();
```

Comment:
Remember to give this [a widget that will allow it to be set by the driver](https://github.wpilib.org/allwpilib/docs/release/java/edu/wpi/first/wpilibj/shuffleboard/BuiltInWidgets.html).